README.md

```diff
@@ -68,3 +68,32 @@ Go to admin/ and sign in with the admin password you just set.
 The repository unit tests and Docker images are built by CircleCI at https://circleci.com/gh/ministryofjustice/laa-legal-adviser-api.
 
 Deployment can be triggered via https://ci.service.dsd.io/job/DEPLOY-laalaa.
+
+## Releasing
+
+### Releasing to non-production
+
+> Currently, `staging` is the only non-production environment.
+
+1. Wait for [the Docker build to complete on CircleCI](https://circleci.com/gh/ministryofjustice/laa-legal-adviser-api) for the feature branch.
+1. Copy the `feature_branch.<sha>` reference from the `build` job's "Push Docker image" step. Eg:
+    ```
+    Pushing tag for rev [1ad776954b2e] on {https://registry.service.dsd.io/v1/repositories/laalaa/tags/dependabot-pip-django-filter-2.0.0.7243223}}
+    ```
+1. [Deploy `feature_branch.<sha>`](https://ci.service.dsd.io/view/LaaLaa/job/DEPLOY-laalaa/build?delay=0sec).
+    * `ENVIRONMENT` is the target environment, select "staging".
+    * DEPLOY_BRANCH is the [deploy repo's](https://github.com/ministryofjustice/laalaa-deploy) default branch name, usually master.
+    * VERSION is the branch that needs to be released plus a specific 7-character prefix of the Git SHA. (`dependabot-pip-django-filter-2.0.0.7243223` for the above example).
```

Comment:
_consistency nitpick_
The bottom 2 variables are plain text while the above is quoted with backticks. Should we make them consistent?